01_01_Basic_Rules.txt

```diff
@@ -346,23 +293,76 @@ table, below.
 | 9-10  | Nothing bad happens... this time                                                      |
 +-------+---------------------------------------------------------------------------------------+
 
+===== Secondary Stats =====	
 
-===== Secondary Stat Details =====
+	Your Character's Secondary stats are each equal to one of your Primary 
+Stats, but no single Primary Stat can be used for two of your Secondary Stats.
 
 Movement Speed:
-	Your speed in meters, for every two actions. Use (Movement Speed)/2 for one 
-action (drop the remainder). For more information, see topic "Movement and 
-Sprinting", below.
+	Your Movement Speed is how fast your character can move every two Actions in
+combat, measured in meters. Use (Movement Speed)/2 for one Action and drop the 
+remainder. For more detail on movement in combat, see the "Movement and 

```

Comment:
New Issue(?): Oops. I haven't been dropping the remainder before multiplying when moving for two actions.
Honestly I like my way better.

Reply:
Fortunately, the rules do not mandate dropping any remainders when moving 2 actions. The base unit is Move Speed per 2 Actions, and the division only happens when moving for exactly 1 Action.